test/rest.test.js

```diff
@@ -110,7 +110,7 @@ describe('strong-remoting-rest', function() {
         }
       );
 
-      objects.options.errorHandler.handler = function(err, req, res, next) {
+      objects.options.strongErrorHandler.handler = function(err, req, res, next) {
```

Comment:
Ditto, this line should not need to be changed, the old API should continue to work in the new version.